documentation/src/main/asciidoc/topics/proc_adding_spring_cache.adoc

```diff
@@ -1,7 +1,7 @@
 [id='adding-spring-cache_{context}']
-= Adding Spring Cache support
+= Adding Spring cache support
 
-To add caching support to your application, with the following annotations:
+To add caching support to your application, use the following annotations:
 
 * `@Cacheable` adds entries to the cache.
 * `@CacheEvict` removes entries from the cache.
```

Comment:
We should drop this part because it is redundant and belongs in the concept topic.